jina/logging/logger.py

```diff
@@ -82,14 +82,19 @@ class SysLogHandlerWrapper(logging.handlers.SysLogHandler):
 class JinaLogger:
     supported = {'FileHandler', 'StreamHandler', 'SysLogHandler', 'FluentHandler'}
 
-    def __init__(self, context: str, log_config: Optional[str] = None, **kwargs):
+    def __init__(self, context: str,
+                 log_config: Optional[str] = os.getenv('JINA_LOG_CONFIG',
+                                                       resource_filename('jina', '/'.join(
+                                                           ('resources', 'logging.default.yml')))),
+                 group_id: Optional[str] = os.getenv('JINA_LOGGING_ID', None), **kwargs):
```

Comment:
Maybe move that into the method. It looks quite ugly, right?

Reply:
It is ugly but is clearly a default no?

Reply:
Yes. But you could do

```python
    def __init__(self, 
                 context: str,
                 log_config: Optional[str] = None,
                 group_id: Optional[str] = None, 
                  **kwargs):
              if log_config is None:
                      log_config =  os.getenv('JINA_LOG_CONFIG',
                                                       resource_filename('jina', '/'.join(
                                                           ('resources', 'logging.default.yml'))))
                [and same for the other]
```

Just a matter of preference

Reply:
Yes yes is what I changed partially, I will change back